server/proxy/tcpproxy/userspace.go

```diff
@@ -69,13 +69,31 @@ type TCPProxy struct {
 	pickCount int // for round robin
 }
 
+// The parameter host is returned by net.SplitHostPort previously,
+// so it must be a valid host. This function is only to check whether
+// it's an IPv6 IP address.
+func isIPv6(host string) bool {
+	for i := 0; i < len(host); i++ {
```

Comment:
This code just looks for ":" in string. Why not use strings.Index(host, ":") != -1

Reply:
Good point. String.IndexByte is better. Resolved.